src/lib/inputs/Input.svelte

```diff
@@ -89,9 +93,13 @@
 <div class={containerClassString}>
   {#if label}
     {#if labelIn}
-      <label class="label-in" for={$$restProps.id}>{$$restProps.id}</label>
+      <label class="label-in" for={$$restProps.id} style="color: {labelColor}"
+        >{$$restProps.id}</label
+      >
     {:else}
-      <label class="label" for={$$restProps.id}>{$$restProps.id}</label>
+      <label class="label" for={$$restProps.id} style="color: {labelColor}"
+        >{$$restProps.id}</label
+      >
     {/if}
   {/if}
   <input
```

Comment:
if there are situations where the user can totally omit having a label you should consider adding an ariaLabel?:string to the interface

Reply:
So in that case should I always require an id? If I did that, I could just always set that ID to either the aria-label || the label itself?